float64 comm_pos_x
float64 comm_pos_y
float64 comm_pos_z
float64 comm_rot_r
float64 comm_rot_p
float64 comm_rot_y

#ftag2test/DisplayTag displayer

#int32 frameID

string command
string fsm
int32 pos_count
int32 total_img_count
int32 num_tags_in_pose_count
int32 num_detections_curr_img
int32 num_failed_det_curr_img
int32 num_det_in_pose
int32 num_failed_det_in_pose

bool failed_image
bool failed_pose

string tag_payload
string tag_family
int32 num_tag_family
